src/components/Input/BasicInput.tsx

```diff
@@ -11,45 +11,106 @@ interface BasicInputProps {
   placeholder?: string;
   commentBackground?: boolean;
 
-  value: string;
-  onChange: (e: ChangeEvent<HTMLInputElement>) => void;
+  value?: string;
+  defaultValue?: string;
+  onChange?: (e: ChangeEvent<HTMLInputElement>) => void;
 
   showClear?: boolean;
   onClear?: () => void;
   showEyeIcon?: boolean;
   showTextLength?: boolean;
   isInvalid?: boolean;
+  isValidating?: boolean;
+  validationMessage?: string;
   minLength?: number;
   maxLength?: number;
   errorMessage?: string;
   successMessage?: string;
   className?: string;
+  ref?: Ref<HTMLInputElement>;
 }
 
 const BasicInput = ({
   type,
   label,
   placeholder,
   value,
+  defaultValue,
   onChange,
   showClear = false,
   onClear,
   showEyeIcon = false,
   showTextLength = false,
   isInvalid = false,
+  isValidating = false,
+  validationMessage,
   commentBackground = false,
   minLength,
   maxLength,
   errorMessage,
   successMessage,
   className,
+  ref,
+  ...props
 }: BasicInputProps) => {
   const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const [uncontrolledLength, setUncontrolledLength] = useState(
+    defaultValue?.length || 0,
```

Comment:
```suggestion
    defaultValue?.length ?? 0,
```

`defaultValue`가 `undefined`면 0을 기본값으로 주는 것도 좋을 것 같아용